src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -3579,3 +3586,62 @@ void MavlinkReceiver::stop()
 	_should_exit.store(true);
 	pthread_join(_thread, nullptr);
 }
+
+void MavlinkReceiver::parse_for_ok(int nread, const char *buf)
+{
+	if (nread < 1) {
+		return;
+	}
+
+	for (int i = 0; i < nread; ++i) {
+		switch (_parse_state) {
+		case OkParseState::None:
+			// Nothing to do.
+			break;
```

Comment:
might as well break from the for loop here too, right?

Reply:
Right!